Cake.Recipe/Content/build.cake

```diff
@@ -555,6 +459,7 @@ public class Builder
 
     private static void SetupTasks(bool isDotNetCoreBuild)
     {
+        var testTasks = new List<string>();
```

Comment:
@AdmiringWorm what was this added for?  Don't see it being used anywhere.

Reply:
I added it initially to make it simpler to reference the unit test project, this didn't turn out how I wanted though.
I just forgot to remove it afterward, will do it on next update (currently working on the integration tests to make them succeed, problems with OpenCover)